openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx

```diff
@@ -332,6 +335,35 @@ const DashboardDetails = ({
       tagUpdateHandler(updatedDashboard);
     }
   };
+
+  const handleRestoreTable = async () => {
+    const data: CreateDashboard = {
+      charts: dashboardDetails.charts,
+      dashboardUrl: dashboardDetails.dashboardUrl,
+      description: dashboardDetails.description,
+      displayName: dashboardDetails.displayName,
+      extension: dashboardDetails.extension,
+      name: dashboardDetails.name,
+      owner: dashboardDetails.owner,
+      service: dashboardDetails.service,
+      tags: dashboardDetails.tags,
+    };
+
+    try {
+      await restoreDashboard(data);
+      showSuccessToast(
+        jsonData['api-success-messages']['restore-table-success'],
+        2000
+      );
```

Comment:
@Ashish8689 , This is a dashboard entity so the message should be related to the dashboard entity, not the table entity.